examples/samplecontainer/contents/cr-service.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: cr-service
+spec:
+  ports:
+  - protocol: TCP
+    port: 443
+---
+kind: Endpoints
+apiVersion: v1
+metadata:
+  name: cr-service
+subsets:
+  - addresses:
+      - ip: 172.18.0.2  #BAD
```

Comment:
Why #BAD?

Reply:
Because this relies on the network created by docker, which may not have this IP address.

I figured out how to fix this, though, and to make it not depend on the Docker network.